indico/web/client/js/jquery/index.js

```diff
@@ -52,6 +52,10 @@ import './utils/forms';
 import './utils/misc';
 import './utils/routing';
 import './utils/sortablelist';
+import 'selectize';
+import 'selectize/dist/css/selectize.css';
+import 'selectize/dist/css/selectize.default.css';
```

Comment:
I'd include this earlier e.g. before the `./widgets` import since that's where we also import all the other jquery stuff